inst/shiny/global.R

```diff
@@ -45,6 +45,13 @@ var_filter <- c("Survey Name" = "SURVEY_NAM",
                 "Survey Year" = "SURVEY_YEA",
                 "Survey ID" = "SURVEY_ID")
 
+# Columns which can be subsetted (during PI exploration/summaries)
+var_subset <- c("MMU_ID", "SURVEY_NAM", "YT_REGION", "SURVEY_YEA", "SURVEY_ID",
```

Comment:
These are the meta data variables available to use as subset columns in the PI subsetting... are there any here that should not be? Are there any which are missing?

Reply:
Looks like there is also a `"S_SET_ID"` variant. Making it as a global variable is handy b/c then it is easy to update based on feedback.